### PR TITLE
No http

### PR DESCRIPTION
**Pull request by 154pinkchairs**
Minor refactor, but also includes the changes included in my previous PR. src/pages/options/widgets/services.ejs now should default to https for custom instance placeholders, as well as the resources fetched from w3.org. HTTP should be strictly avoided for clearnet.

### CODE REVIEW

**Comment by EdwardLangdon** [comment]
> HTTPS should be strictly avoided for clearnet.

You mean HTTP ?

File: src/pages/options/index.html [hunk 1]
```diff
@@ -200,7 +203,7 @@ <h4 data-localise="__MSG_exceptions__"></h4>
     </div>
   </form>
   <div class="checklist" id="exceptions-custom-checklist"></div>
-  <div class="buttons buttons-inline"><a class="button button-inline" id="update-instances">
+  <div class="butto/ns buttons-inline"><a class="button button-inline" id="update-instances">
```
Comment (AlyoshaVasilieva) [inline src/pages/options/index.html:206]: errant slash?

File: src/pages/options/index.html [hunk 2]
```diff
@@ -3310,7 +3313,7 @@ <h4 data-localise="__MSG_frontend__">Frontend</h4>
       <div class="some-block option-block">
         <h4 data-localise="__MSG_defaultInstances__">Default Instances</h4>
       </div>
-      <div class="checklist"></div>
+      <div class="checklist"></div/>
```
Comment (AlyoshaVasilieva) [inline src/pages/options/index.html:3316]: think `</div/>` is also invalid

**Comment by ManeraKai** [comment]
> now should default to https for custom instance placeholders

Some people want to use normal http if for example the server is a localhost one. I would not interfere that.

**Comment by ManeraKai** [comment]
We also removed Latency Tests, and Unify. Thanks for pointing out about the w3.org though, I corrected them.

**Pull request closed without merging**